src/Libraries/DynamoUnits/Units.csproj

```diff
@@ -10,10 +10,18 @@
     <AssemblyName>DynamoUnits</AssemblyName>
     <DocumentationFile>$(OutputPath)\DynamoUnits.xml</DocumentationFile>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'net48' ">
```

Comment:
I'm confused by this change, it looks like net48 no longer references forge units?